site/content/internal/rd-teams.md

```diff
@@ -9,112 +9,135 @@ weight: 110
 
 * Corey Hulen - CTO
 * Chris Overton - VP Engineering
-* Lindsay Brock - Head of Product
+* Chandar Venkataraman - CPO
 
-## Mobile Team
+## Feature Teams
+Feature teams are full-stack teams owning collections of features within the Mattermost product.
 
-* Elias Nahum - Eng Lead
-* Miguel Alatzar - Engineer
-* Amit Uttam - Engineer
-* Eric Sethna - PM
-* Matt Birtch - UX Designer
+### Enterprise Features Team
 
-## Apps Team
+* Scott Bishel - Engineering Lead
+* Martin Kraft - Engineeer
+* Hossein Ahmadian - Engineer
+* Anurag Shivarathri - Engineer
+* Dennis Kittrell - Product Manager
+* Furqan Malik - QA SDET
+* Michael Gamble - UX Designer
+* Justine Geffen - Technical Writer
 
-* Dean Whillier - Eng Lead
-* Harrison Healey - Engineer
-* Sudheer Timmaraju - Engineer
-* Devin Binnie - Engineer
-* Guillermo Vayá - Engineer
-* Nevy Angelova - Engineer
+### Core Features Team
+
+* Scott Bishel - Engineering Lead
+* Jesus Espino - Engineer
+* Eli Yukelzon - Engineer
+* Ashish Bhate - Engineer
 * Caleb Roseland - Engineer
-* Eric Sethna - PM (Driving NPS)
-* Adam Clarkson - PM (Driving Early Retention)
+* Eric Sethna - Product Manager
+* Ogi Marusic - QA
 * Andrew Brown - UX Designer
-
-## Cloud Team
-
-* Joram Wilander - Eng Lead
-* Carlos Panato - SRE
-* Gabe Jackson - Engineer
-* Stylianos Rigas - SRE
-* Ian Whitlock - Engineer
-* Gabriel Sagula - Engineer
-* Angelos Kyratzakos - SRE
-* Katie Wiersgalla - PM (Cloud)
-* Eric Sadur - PM (Self-serve)
-* Matt Birtch - UX Designer
 * Justine Geffen - Technical Writer
 
-## Build & Infrastructure Team
-
-* Jason Deland - Engineer Lead
-* Elisabeth Kulzer - Build Engineer
+### Integrations Team
 
-## Integrations Team
-
-* Lev Brouk - Eng Lead
+* Catalin Tomai - Engineering Lead
+* Lev Brouk - Engineer
 * Jason Frerich - Engineer
 * Michael Kochell - Engineer
 * Shota Gvinepadze - Engineer
 * Ben Schumacher - Engineer
 * Daniel Espino Garcia - Engineer
-* Aaron Rothschild - PM
+* Aaron Rothschild - Product Manager
 * Dylan Haussermann - QA
-* Asaad Mahmood - UX Designer
 
-## Platform Team
+### Workflows Team
 
-* George Goldberg - Eng Lead
-* Jesus Espino - Engineer
+* Jesse Hallam - Engineering Lead
+* Christopher Speller - Engineer
+* Christopher Poile - Engineer
+* Alejandro García Montoro - Engineer
+* Ian Tao - Product Manager
+* Prapti Shrestha - QA SDET
+
+### Cloud Features Team
+
+* Maria Nuñez - Engineering Lead
+* Mario de Frutos - Engineer
+* Nick Misasi - Engineer
+* Eric Sadur - Product Manager
```

Comment:
```suggestion
* Adam Clarkson - Product Manager
```

Reply:
@jwilander change incorporated. Can you update the internal spreadsheet too?

Reply:
It's got both Adam and Eric on the spreadsheet already since they both interact with the team but Adam is the dedicated PM for the team